lib/Analysis/BlockFrequencyInfoImpl.cpp

```diff
@@ -547,8 +547,10 @@ BlockFrequencyInfoImplBase::getProfileCountFromFreq(const Function &F,
   // Use 128 bit APInt to do the arithmetic to avoid overflow.
   APInt BlockCount(128, EntryCount.getValue());
   APInt BlockFreq(128, Freq);
-  APInt EntryFreq(128, getEntryFreq());
+  auto EntryFreqTemp = getEntryFreq();
```

Comment:
Nit. I prefer naming this as EntryFreq and the APInt as EntryFreqAPInt or something like that. I'll leave it to you.